docs/apache-airflow/concepts/timetable.rst

```diff
@@ -48,3 +48,58 @@ the schedule. Some examples are:
 As such, Airflow allows for custom timetables to be written in plugins and used by
 DAGs. An example demonstrating a custom timetable can be found in the
 :doc:`/howto/timetable` how-to guide.
+
+Built In Timetables
+-------------------
+
+Airflow comes with several common timetables built in to cover the most common use cases. Additional timetables
+may be available in provider packages.
```

Comment:
```suggestion
may be available in plugins.
```

They come from plugins, not providers, right?

Reply:
Yes, that's true. I will correct it.